operatorhub/openshift/config.yaml

```diff
@@ -204,6 +204,14 @@ image-substitutions:
         containerName: openshift-pipelines-operator-lifecycle
         envKeys:
           - IMAGE_HUB_TEKTON_HUB_UI
+- image: registry.redhat.io/openshift-pipelines/pipelines-manual-approval-gate-rhel8@
+  replaceLocations:
+    envTargets:
+      - deploymentName: openshift-pipelines-operator
+        containerName: openshift-pipelines-operator-lifecycle
+        envKeys:
+          - IMAGE_MAG_TEKTON_TASKGROUP_CONTROLLER
+          - IMAGE_MAG_MANUAL_APPROVAL
```

Comment:
@PuneetPunamiya Why do we use two type of environment keys for ManualApprovalGate image?

Reply:
We have two deployments in manual approval, one for controller and one for webhook

Reply:
thanks @PuneetPunamiya 
however the container name looks bit odd :)
* `tekton-taskgroup-controller` => ManualApprovalGate Controller
* `manual-approval` => ManualApprovalGate Webhook